SQL/advanced.sql

```diff
@@ -383,7 +383,7 @@ CREATE VIEW booking_costs_today AS (
 STORED FUNCTIONS
 */
 
--- passing parameters
+/* #1: passing parameters */
```

Comment:
you can also create folder SQL/stored_procedures and put each task in separate file :) will be super convenient :)